packages/turf-isobands/index.js

```diff
@@ -0,0 +1,269 @@
+// https://github.com/RaumZeit/MarchingSquares.js
+var MarchingSquaresJS = require('./marchingsquares-isobands.min');
+
+var turfHelpers = require('@turf/helpers');
+var turfFeaturecollection = turfHelpers.featureCollection;
+var turfPolygon = turfHelpers.polygon;
+var turfMultiPolygon = turfHelpers.multiPolygon;
+var turfExplode = require('@turf/explode');
+var turfInside = require('@turf/inside');
+var turfArea = require('@turf/area');
+
+/*******************************************************************
```

Comment:
Follow the syntax of all other JSDocs Turf modules, simply using `/**` is enough.